vars/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+# vars file for test-role
```

Comment:
This might cause some parsing issues if the file is blank, since ansible expects this to be populated. Saw a similar issue in defaults/*.yml, perhaps it's just an issues with defaults

Reply:
What conditions do I need to test under to check that? Travis didn't notice any parsing problems and neither did I.

Reply:
I've noticed that too, but in my case Travis caught it. It's a thing with the `ansible-playbook --syntax-check` that will fail.

Reply:
Must only apply to defaults? https://github.com/infrawatch/collectd-config-ansible-role/blob/master/vars/main.yml

Reply:
Ahhh yes, that does sound familiar now! I think you're good then.